Monika After Story/game/script-story-events.rpy

```diff
@@ -1471,6 +1471,86 @@ label mas_steam_install_detected:
     m 5esu "I'd really appreciate it if you would do that for me."
     return
 
+default persistent._mas_pm_has_rpy = None
+init 1 python:
+    import store
+    import store.mas_ics as mas_ics
+    # setup the docking station to handle the rpys
+    rpyCheckStation = store.MASDockingStation(mas_ics.game_folder)
+
+    listRpy = MASDockingStation.getPackageList(rpyCheckStation,".rpy")
+
+    if len(listRpy) != 0 and persistent._mas_pm_has_rpy is not True:
+        pushEvent("monika_rpy_files")
+
+
+init 5 python:
+
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_rpy_files",
+            rules={"no unlock": None}
```

Comment:
dont use `rules` here, since this is not a pool topic it will not unlock right away.